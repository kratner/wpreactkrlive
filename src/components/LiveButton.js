import React from "react";
import { Button } from "@material-ui/core";

export default function LiveButton(props) {
  return <Button {...props} />;
}
