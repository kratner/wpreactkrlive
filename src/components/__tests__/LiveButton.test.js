import React from "react";
import { shallow } from "enzyme";
import LiveButton from "../LiveButton";

describe("LiveButton", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<LiveButton />);
    expect(wrapper).toMatchSnapshot();
    // On the first run of this test, Jest will generate a snapshot file automatically.
  });
});
