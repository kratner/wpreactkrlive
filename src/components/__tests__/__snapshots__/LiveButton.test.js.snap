// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiveButton renders correctly 1`] = `ShallowWrapper {}`;
